.gitattributes

```diff
@@ -0,0 +1,28 @@
+# Declare the different kinds of text files, and make sure they always have the same line endings on checkout (unix)
+*.txt text eol=lf
+*.md text eol=lf
+*.cs text eol=lf
+*.js text eol=lf
+*.ts text eol=lf
+*.html text eol=lf
+*.css text eol=lf
+*.scss text eol=lf
+*.cshtml text eol=lf
+*.csproj text eol=lf
+*.csproj.DotSettings text eol=lf
+*.aspx text eol=lf
+*.ascx text eol=lf
+*.vm text eol=lf
+*.Master text eol=lf
+*.xprojtext eol=lf
+*.xproj.DotSettings text eol=lf
+*.xproj.user text eol=lf
+*.json text eol=lf
+*.xml text eol=lf
+*.config text eol=lf
+*.config.template text eol=lf
+*.msbuild text eol=lf
+
+*.sln text eol=lf
+*.vcproj text eol=lf
+*.vcxproj text eol=lf
```

Comment:
This file has nothing to do with the PR.

Reply:
It prevented me from committing crlf, which I almost did by accident without knowing it.

Reply:
I recommend you fix your local git settings. :D

Though in principle I'm not against including a file like this in the repo.

Reply:
No can do, because at work we use crlf. I'm not aware of a repo-local solution for this, that doesn't involve tracking a .gitattributes file.